src/guake/about.py

```diff
@@ -43,5 +43,5 @@ def __init__(self):
         img = gtk.gdk.pixbuf_new_from_file(ipath)
         dialog.set_property('logo', img)
 
-        dialog.set_name('Guake Terminal')
+        dialog.set_name(_('Guake Terminal'))
```

Comment:
beware this won't work, you need to include '_' (https://travis-ci.org/Guake/guake/builds/62825994)